### ISSUE TITLE
Indicate if buttons are pressed or if menus are expanded

### ISSUE THREAD

**Issue by nvdaes** [readium/readium-desktop#1005]
Thorium has graphics like navigation which can be pressed to expand or collapse elements, and this is not announced to screen readers users. Use aria or other procedures to show this

**Comment by danielweck** [comment]
Could you please explain which button has a confusing collapsed / expanded state? Thank you!

**Comment by nvdaes** [comment]
> danielweck commented 2 hours ago 
Could you please explain which button has a confusing collapsed / expanded state? Thank you!

I'm talking about navigation button, table of contents button and so on. They can be pressed to expand and collapse the navigation, table of contents list, etc., but this is not announced for screen readers.

**Comment by nvdaes** [comment]
Expanding on this, I wanted to test this code, but unfortunately seems books cannot be imported in Thorium running `npm start`.

```
                            <li
                            {...(this.props.menuOpen && {style: {backgroundColor: "rgb(193, 193, 193)"}})}
                            >
                                <button
                                    className={styles.menu_button}
aria-pressed={this.props.menuOpen}                                    onClick={this.props.handleMenuClick.bind(this)}
                                    ref={this.navigationMenuButtonRef}
```
This is useful since, for example, after pressing control+n, then Enter, the navigation menu is not opened. In this case, Enter must be pressed twice.
Also, sometimes we know that bookmarks have been saved, and it is useful to check the state of the bookmarks button to avoid confusions if we don't see bookmarks pressing down arrow.
I am not sure if the suggested code could be useful since I haven't been able to test it without importing a book.

**Comment by danielweck** [comment]
`npm start` (or alternatively, `npm run start`) launches Thorium in "production" mode, which does not expose the web inspectors / debuggers (which can be useful in screen reader scenarios, as these extra windows add a lot of noise to the experience, potentially slowing down the computer quite a bit too).

However, as far as I know (i.e. this is tested regularly) it is of course possible to import publications. As developers, we tend to launch the app in "development" mode which is `npm run start:dev`, could you please try this and report back to us?

Failing that, there probably is another problem stemming back from how you compiled Thorium in the first place. For example, the production mode requires a separate initialization step immediately after `npm install`, which is `npm run __postinstall`.

Also, when working from the develop branch, it is important to regularly update the package dependencies in order to stay synchronized with the main repository. I would suggest the following commands:
* `rm -rf node_modules` (to completely remove the `node_modules` folder)
* `git diff package-lock.json` (to verify that there are NO changes in your local NPM lock file, which sometimes happens on Windows ... if so, just invoke `git checkout package-lock.json` to restore the authoritative codebase version)
* `npm install`
* `npm run start:dev` (or alternatively, first run `npm run __postinstall` once, then repeat `npm run start` as needed to launch the app)

**Comment by danielweck** [comment]
> This is useful since, for example, after pressing control+n, then Enter, the navigation menu is not opened. In this case, Enter must be pressed twice.

CTRL-N automatically opens the navigation menu and places the keyboard focus on the dedicated button icon above it. Therefore, you must keyboard-tab in order to enter the actual slide-out menu (which is visually located on the right hand side, by the way).

**Comment by danielweck** [comment]
Update: I have just fixed a bug in the `develop` branch which may explain why some publications were failing to import or load. Please `git pull` to update your local copy of the repository, or alternatively you may use the automated build available from this link https://github.com/readium/readium-desktop/releases/tag/latest-windows
(the app installer is currently building, it should be ready within the next 30mn)

**Comment by danielweck** [comment]
Hello Noelia, thank you for suggesting `aria-pressed`, it works well with Voice Over. Could you please verify with NVDA?
I added this technique in multiple places in the reader and the library windows, wherever a UI link or button have on/off toggled states.

**Comment by nvdaes** [comment]
@danielweck , aria-pressed works right with NVDA! This is wonderful, thanks.

**Comment by nvdaes** [comment]
About importing books, now I can do it running `npm run start:dev`.
When I try starting the app in production mode after running `npm run __postinstall`, I get an error related to Windows platform.

< Error  diálogo  A JavaScript error occurred in the main process
Uncaught Exception:
Error: No native build was found for platform=win32 

PS: I am not closing issues in case you want to add something. Please feel free to close whatever you consider. aria-pressed works for me, as well as moving to TOC, bookmarks and page numbers with NVDA.
